lib/cartopy/tests/crs/test_interrupted_goode_homolosine.py

```diff
@@ -60,11 +60,10 @@ def test_central_longitude(emphasis, lon):
     elif emphasis == "ocean":
         check_proj_params("igh_o", igh, other_args)
 
-    assert_almost_equal(
+    assert_allclose(
         np.array(igh.x_limits),
         [-20037508.3427892, 20037508.3427892],
-        decimal=5,
```

Comment:
Not sure about dropping this, as the default `rtol` is 1e-7, not 1e-5, but I forget the exact difference between the two asserts.

Reply:
IMO, these tolerances are all playing whack-a-mole with PROJ updates that are legitimate corrections on the projection math. I think the errors we want to avoid are the egregious ones, which this test will still catch.

Reply:
Actually, I didn't realize it was only these two files; what you say is probably true for them.